CalibCalorimetry/EcalLaserCorrection/src/EcalLaserDbService.cc

```diff
@@ -164,6 +170,7 @@ float EcalLaserDbService::getLaserCorrection(DetId const& xid, edm::Timestamp co
   // interpolation
 
   edm::TimeValue_t t = iTime.value();
+  long long t_laser = t;
```

Comment:
For code clarity I suggest to move this line to 201, so the definition of t_laser is grouped in one place.